### ISSUE TITLE
image's alt text is missing

### ISSUE THREAD

**Issue by s-kris** [JiHong88/SunEditor#601]
Hello!

When uploading an image, we can attach alt text which is getting saved to database.
But when the post is loaded from database the alt text is missing.

Not sure if this is caused by react wrapper, hence linking both: https://github.com/mkhstar/suneditor-react/issues/161

Thank you

**Comment by JiHong88** [comment]
@s-kris This issue is not reproduced.
Please tell me in detail the options when creating the editor and how to test it.

**Comment by JiHong88** [comment]
"alt" was missing from the whitelist.
If you add the option below, it will work before fix.
```javascript
attributesWhitelist: {
  img: 'alt',
}
```

**Comment by s-kris** [comment]
@JiHong88 thanks for the quick response. 
I tested in both versions. It's working :)

**Comment by JiHong88** [comment]
The [2.36.0](https://github.com/JiHong88/SunEditor/releases/tag/2.36.0) version has been updated.
If this issue has not been resolved, please reopen this issue.
Thank you.
